ksql-package/src/assembly/package.xml

```diff
@@ -2,34 +2,59 @@
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2
           http://maven.apache.org/xsd/assembly-1.1.2.xsd">
-    <!-- Assembles a package that can run ksql -->
+    <!-- Assembles a package that can run ksql given dependency packages. This is used to
+         construct packages OS packages where underlying libraries, like rest-utils, can be shared
+         -->
     <id>package</id>
     <formats>
         <format>dir</format>
         <format>tgz</format>
-        <format>zip</format>
     </formats>
     <includeBaseDirectory>true</includeBaseDirectory>
     <baseDirectory>ksql</baseDirectory>
     <fileSets>
         <fileSet>
             <directory>${project.parent.basedir}</directory>
-            <outputDirectory></outputDirectory>
+            <outputDirectory>share/doc/ksql/</outputDirectory>
             <includes>
+                <include>version.txt</include>
+                <include>README*</include>
                 <include>LICENSE*</include>
+                <include>NOTICE*</include>
                 <include>licenses/</include>
                 <include>notices/</include>
-                <include>bin/</include>
-                <include>config/</include>
```

Comment:
It should work if you add this back in, and get rid of lines 35-41. Or change etc/ksql on line 37 to config.